src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -780,6 +843,37 @@ int sd_dhcp_lease_load(sd_dhcp_lease **ret, const char *lease_file) {
                            "ROUTES", &routes,
                            "CLIENTID", &client_id_hex,
                            "VENDOR_SPECIFIC", &vendor_specific_hex,
+                           "OPTION_224", &options[DHCP_OPTION_PRIVATE_0 - DHCP_OPTION_PRIVATE_START],
```

Comment:
I'm not thrilled with this but I don't have a better way to write it. I tried generating the variadic list at runtime, but strv isn't well suited for this (I would need a `char **` instead of a `char *` for the type). I also considered using [x macros](https://en.wikipedia.org/wiki/X_Macro), but I haven't seen them used elsewhere in the codebase. I'm open to suggestions.